src/bio/smd/smd_pool.c

```diff
@@ -275,7 +302,7 @@ smd_pool_list(d_list_t *pool_list, int *pools)
 		return 0; /* There is no NVMe, smd will not be initialized */
 
 	smd_db_lock();
-	rc = smd_db_traverse(TABLE_POOL, smd_pool_list_cb, &td);
+	rc = smd_db_traverse(TABLE_POOLS[SMD_DEV_TYPE_DATA], smd_pool_list_cb, &td);
```

Comment:
It assumes smd_pool_list() supports only DATA device? if so, the changes in smd_pool_list_cb() won't be necessary.

Reply:
Ok, sounds reasonable to me.